daisy_workflows/image_import/inspection/src/boot_inspect/inspection.py

```diff
@@ -117,6 +117,42 @@ def inspect_device(g, device: str) -> model.InspectionResults:
   )
 
 
+def inspect_boot_loader(g) -> (bool, bool, str):
+  """Finds boot-loader properties for the device using offline inspection.
```

Comment:
A tuple is difficult to understand and test, especially when there are more than two values. In terms of unit testing, I suspect having one method per inspection will be easier. You could also consider returning an object with fields.

Reply:
done